content/documentation/modelHubApproach/content.md

```diff
@@ -0,0 +1,733 @@
++++
+fragment = "content"
+title = "Model Hub Approach"
+weight = 160
+[sidebar]
+  sticky = true
++++
+
+# Revolutionize Model Management with the Model Hub Approach
```

Comment:
Same here, I'd remove this heading (also a bit too flashy for my taste) and push all headings below to H3.